docker/include/network/scripts/start_networking

```diff
@@ -15,7 +15,7 @@ while ! ip link show $EXT_INTF; do
     sleep 5
 done
 
-if [[ $EXT_MAC ]]; then
+if [[ -n $EXT_MAC && $EXT_MAC != None ]]; then
```

Comment:
I don't see where this is being used and why is it possible for EXT_MAC be None?

Reply:
It gets converted from ext_mac to EXT_MAC in the code -- (in base_gateway.py _get_env_vars() does the case conversion). So, it's set in the configuration as ext_mac, and then gets passed along as EXT_MAC (like a few other parameters). It can be None because somewhere in there python is automatically converting a None value to "None". The alternate fix would be to add an if statement in the code that says "if not value then don't add the env var setting" or sets the value to "". Either way, there's an if statement somewhere, just not sure where (I was going with the principle of the most-focused change, but there are other principles to follow).  What do you think?

Reply:
I think we can follow https://github.com/faucetsdn/daq/blob/04967711fc58c3101c2449d09dbc9dcef95df21e/daq/test_modules/external_module.py#L110 
to convert None to "" and check here [ ! -z $EXT_MAC ]